guillotina_amqp/tests/fixtures.py

```diff
@@ -11,29 +26,76 @@ def base_settings_configurator(settings):
         ])
     else:
         settings['applications'] = ['guillotina_amqp', 'guillotina_amqp.tests.package']
-    settings['amqp'] = {
-        "connection_factory": "guillotina_amqp.tests.mocks.amqp_connection_factory",
-        "host": "localhost",
-        "port": 5673,
-        "login": "guest",
-        "password": "guest",
-        "vhost": "/",
-        "heartbeat": 800,
-        "exchange": "",
-        "queue": "guillotina",
-        "persistent_manager": "memory"
-    }
+    settings['amqp'] = base_amqp_settings
 
 
 testing.configure_with(base_settings_configurator)
 
 
 @pytest.fixture('function')
-def amqp_worker(loop):
+def amqp_worker(loop, rabbitmq_container):
+    # Create worker
     _worker = Worker(loop=loop)
+    _worker.update_status_interval = 2
     loop.run_until_complete(_worker.start())
+
     yield _worker
+
+    # Tear down worker
     for conn in [v for v in app_settings['amqp'].get('connections', []).values()]:
         loop.run_until_complete(conn['protocol'].close())
     _worker.cancel()
     app_settings['amqp']['connections'] = {}
+
+
+@pytest.fixture('function', params=[
+    {'redis_up': True},
```

Comment:
Clever. I was not familiar with this!

Reply:
yeah it's pretty handy! same tests are ran twice